vnext/Microsoft.ReactNative/Views/Image/ReactImage.cpp

```diff
@@ -140,7 +140,11 @@ void ReactImage::Source(ReactImageSource source) {
     if (((scheme == L"http") || (scheme == L"https")) && !source.headers.empty()) {
       source.sourceType = ImageSourceType::Download;
     } else if (scheme == L"data") {
-      source.sourceType = ImageSourceType::InlineData;
+      if (source.uri.find("image/svg") != std::string::npos) {
```

Comment:
should there be some more parsing of what the url pieces are? e.g. what if they have a url that has that string in it?
`http://foo/bar/image/svg/myimage.svg` ?

Reply:
You should probably just check that the string starts with `data:image/svg+xml;`